.circleci/config.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+jobs:
+  build:
+    working_directory: /go/src/github.com/ahoy-cli/ahoy
+    docker:
+      - image: circleci/golang
+    steps:
+      - checkout
+      - run: git clone https://github.com/bats-core/bats-core.git && cd bats-core && sudo ./install.sh /usr/local && bats --version && cd -
```

Comment:
installing bats from sources. this is easier then installing it from npm, as npm installation into this image requires many more steps